recipes/taproom/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```

Comment:
Could you please remove those unnecessary comments?

Reply:
Done.